packages/ember-views/lib/system/event_dispatcher.js

```diff
@@ -196,12 +196,12 @@ export default EmberObject.extend({
     @param {Object} viewRegistry
   */
   setupHandler(rootElement, event, eventName, viewRegistry) {
-    let self = this;
-
     if (eventName === null) {
       return;
     }
 
+    let self = this;
+
     rootElement.on(`${event}.ember`, '.ember-view', function(evt, triggeringManager) {
```

Comment:
Wouldn't work because of [this](https://github.com/emberjs/ember.js/pull/15019/files#diff-c7b9de937661f47b233bb5e3101436e0R206).